src/shogun/mathematics/Statistics.cpp

```diff
@@ -754,17 +755,43 @@ float64_t CStatistics::inverse_gamma_cdf(float64_t p, float64_t a,
 	return 0;
 }
 
-//float64_t CStatistics::incomplete_beta(float64_t a, float64_t b, float64_t x)
-//{
-//	SG_SERROR("NOT IMPLEMENTED");
-//	return 0;
-//}
+float64_t CStatistics::incomplete_beta(float64_t a, float64_t b, float64_t x)
+{
+	float64_t y=1-x;
+	float64_t result=0;
+	float64_t result2=0;
+	int error_code=0;
+	beta_inc(&a, &b, &x, &y, &result, &result2, &error_code);
+	if (error_code!=0)
+	{
+		switch (error_code)
+		{
+			case 1:
+				SG_SERROR("Params a (%f) or b (%f) cannot be negative!\n", a, b);
```

Comment:
btw we should always print the value of the given numbers as in the REQUIRE elsehwere